dockerfiles/deps/tesseract.sh

```diff
@@ -23,7 +23,8 @@ cd ..
 git clone https://github.com/tesseract-ocr/tessdata.git
 cp tessdata/*.traineddata /usr/local/share/tessdata
 rm -rf /tesseract-ocr /tessdata /autoconf-archive-2017.09.28* /leptonica-1.83.1*
-dnf -y remove opencv* perl-core clang libpng-devel libtiff-devel libwebp-devel libjpeg-devel libjpeg-turbo-devel git-core libtool zlib-devel pkconfig xz
+dnf -y remove opencv* perl-core clang libpng-devel libtiff-devel libwebp-devel libjpeg-devel libjpeg-turbo-devel git-core libtool xz
```

Comment:
there was a typo i pkconfig.
zlib-devel and pkgconfig are system dependencies that were failing to uninstall

Reply:
```
888.8 + rm -rf /tesseract-ocr /tessdata /autoconf-archive-2017.09.28 /autoconf-archive-2017.09.28.tar.xz /leptonica-1.83.1 /leptonica-1.83.1.tar.gz
891.4 + dnf -y remove 'opencv*' perl-core clang libpng-devel libtiff-devel libwebp-devel libjpeg-devel libjpeg-turbo-devel git-core libtool zlib-devel pkconfig xz
891.6 No match for argument: pkconfig
891.7 Error: 
891.7  Problem: The operation would result in removing the following protected packages: grub2-tools-minimal
891.7 (try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```
https://github.com/Unstructured-IO/base-images/actions/runs/9398887844/job/25885201783